### PR TITLE
feature/[PATRO23-51] swagger support

### PR DESCRIPTION
**Pull request by pzajaczkowski**

### CODE REVIEW

File: src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
     <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <Namespace>Intive.Patronage2023.Modules.Example.Api</Namespace>
     <OutputType>Library</OutputType>
+	<GenerateDocumentationFile>True</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup>
+	<ItemGroup>
```
Comment (mateuszmuszynski) [inline src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj:12]: Incorrect indentation

File: src/modules/example/api/Controllers/ExampleController.cs [hunk 2]
```diff
@@ -45,8 +50,20 @@ public async Task<IActionResult> GetExamples(GetExamples request)
 	/// </summary>
 	/// <param name="request">Request.</param>
 	/// <returns>Created Result.</returns>
+	/// <remarks>
+	/// Sample request:
+	///
+	///     POST
+	///     {
+	///        "Name": "Example",
```
Comment (mateuszmuszynski) [inline src/modules/example/api/Controllers/ExampleController.cs:58]: Remove trailing comma

File: src/api/app/Program.cs [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
 
+if (app.Environment.IsDevelopment())
+{
+	app.UseSwagger();
+	app.UseSwaggerUI();
+}
+
 app.UseExampleModule();
 
 app.Run();
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:39]: Put newline as a last character

File: src/api/app/Program.cs [hunk 4]
```diff
@@ -1,11 +1,39 @@
+using System.Reflection;
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:1]: Unused using directive

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

**Review by mateuszmuszynski — APPROVED** [review]

**Review by mateuszgiedzinski — APPROVED** [review]
LGTM

**Pull request merged**